<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectWidget</class>
 <widget class="QWidget" name="SelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="labelEnabled">
       <property name="text">
        <string>Items enabled</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="enabledListWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="disablePushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="enablePushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="labelDisabled">
       <property name="text">
        <string>Items disabled</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="disabledListWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>disablePushButton</sender>
   <signal>clicked()</signal>
   <receiver>SelectWidget</receiver>
   <slot>disableItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enablePushButton</sender>
   <signal>clicked()</signal>
   <receiver>SelectWidget</receiver>
   <slot>enableItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disabledListWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>SelectWidget</receiver>
   <slot>enableItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enabledListWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>SelectWidget</receiver>
   <slot>disableItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableItem()</slot>
  <slot>disableItem()</slot>
 </slots>
</ui>
